shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/ui/FeedingPhotoItem.kt

```diff
@@ -0,0 +1,10 @@
+package com.epmedu.animeal.feeding.presentation.ui
```

Comment:
`ui` package is for composables. 
Please move this data class to `model` package.